frontend/src/components/common/form/FastCurrencyInput.scss

```diff
@@ -2,5 +2,15 @@
   .input-tooltip-wrapper {
     display: flex;
     align-items: center;
+    flex-wrap: wrap;
+    input {
+      width: 85%;
+    }
+    svg {
+      width: 15%;
+    }
+    div {
```

Comment:
The field error wasn't being displayed properly, this ensures that it is displayed on the line after the field.